src/templates/en/2019/base_chapter.html

```diff
@@ -32,6 +32,14 @@
   var el = document.getElementById('num_comments');
   el.innerText = comments + ' ' + (comments == 1 ? 'comment' : 'comments');
 });
+document.addEventListener("keypress", function onPress(event) {
+ 	if (event.key === 'p' || event.key === ',') {
+ 		document.getElementById('previous-chapter').click();
+ 	}
+ 	if (event.key === 'n' || event.key === '.') {
```

Comment:
Should we cater for other versions of these keypresses?

```suggestion
 	if (event.key === 'n' || event.key === 'N' || event.key === '.' || event.key === '>') {
```